packaging/plugins/provision.sh

```diff
@@ -156,8 +152,6 @@ if [ "$#" -gt 9 ]; then
     REL_SUB_USERNAME=${14}
     REL_SUB_PASSWORD=${15}
     DOCKER_BUILDER=true
```

Comment:
So DOCKER_BUILDER is not needed anymore

Reply:
Well, you are right, but building wagon for windows in near future is need to be supported and already handled here but never used by our build system, some client use windows and I you noticed that script you will see that vagrant config for windows still there